src/AuthenticationRequest.js

```diff
@@ -117,7 +116,9 @@ class AuthenticationRequest {
       // render the request URI and terminate the algorithm
       .then(() => {
         let url = new URL(endpoint)
-        url.search = FormUrlEncoded.encode(params)
+
+        // combine with any exists query parameters.
+        Object.entries(params).map(param => url.searchParams.append(param[0], param[1]))
```

Comment:
Looks like `URLSearchParams` encodes it like `response_type=id_token+token` instead of `response_type=id_token%20token`. Should be OK per https://stackoverflow.com/a/1211261 or do we need RFC 1738?